js/node/CMakeLists.txt

```diff
@@ -66,9 +66,17 @@ if(MSVC AND CMAKE_JS_NODELIB_DEF AND CMAKE_JS_NODELIB_TARGET)
   execute_process(COMMAND ${CMAKE_AR} /def:${CMAKE_JS_NODELIB_DEF} /out:${CMAKE_JS_NODELIB_TARGET} ${CMAKE_STATIC_LINKER_FLAGS})
 endif()
 
+if (WIN32)
+  if (${ONNXRUNTIME_GENERATOR} MATCHES "Ninja")
```

Comment:
does ONNXRUNTIME_GENERATOR populated in this CmakeLists.txt?

I think maybe need to update js/node/script/build.ts as well.

Reply:
Yes, it is passed by --cmake_generator "Ninja"